README.md

```diff
@@ -91,24 +91,32 @@ To make Kong Admin accessible from other locations, let's setup Kong itself to p
 From the [admin console](#user-content-admin-console):
 ```bash
 # Create the authenticated `/kong-admin` API, targeting the localhost port:
-curl http://localhost:8001/apis -i -X POST \
-  --data name=kong-admin \
-  --data uris=/kong-admin \
-  --data upstream_url=http://localhost:8001 \
-  --data https_only=true \
-  --data http_if_terminated=true
-curl http://localhost:8001/apis/kong-admin/plugins/ -i -X POST \
+curl http://localhost:8001/services/ -i -X POST \
+  --data 'name=kong-admin' \
+  --data 'protocol=http' \
+  --data 'port=8001' \
+  --data 'host=localhost'
+# Note the Service ID returned in previous response, use it in place of `$SERVICE_ID`.
+curl http://localhost:8001/plugins/ -i -X POST \
   --data 'name=request-size-limiting' \
-  --data "config.allowed_payload_size=8"
-curl http://localhost:8001/apis/kong-admin/plugins/ -i -X POST \
+  --data "config.allowed_payload_size=8" \
+  --data 'service_id=$SERVICE_ID'
+curl http://localhost:8001/plugins/ -i -X POST \
   --data 'name=rate-limiting' \
-  --data "config.second=5"
-curl http://localhost:8001/apis/kong-admin/plugins/ -i -X POST \
+  --data "config.minute=5" \
+  --data 'service_id=$SERVICE_ID'
+curl http://localhost:8001/plugins/ -i -X POST \
   --data 'name=key-auth' \
-  --data "config.hide_credentials=true"
-curl http://localhost:8001/apis/kong-admin/plugins/ -i -X POST \
+  --data "config.hide_credentials=true" \
+  --data 'service_id=$SERVICE_ID'
+curl http://localhost:8001/plugins/ -i -X POST \
   --data 'name=acl' \
-  --data "config.whitelist=kong-admin"
+  --data "config.whitelist=kong-admin" \
+  --data 'service_id=$SERVICE_ID'
+curl http://localhost:8001/routes/ -i -X POST \
+  --data 'paths[]=/kong-admin' \
+  --data 'protocols[]=https' \
+  --data 'service=$SERVICE_ID'
```

Comment:
This line should be:
`--data 'service.id=$SERVICE_ID'`

Reply:
Thanks @Rhadros